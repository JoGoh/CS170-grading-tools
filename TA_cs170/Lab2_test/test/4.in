1 1 21 0
2 1 94 0
3 1 21 0
3 1 1 0
3 1 94 0
4 1 94 0
4 1 20 0
4 1 1 0
4 1 21 0
1 2 10 0
2 2 10 0
3 2 10 0
4 2 10 0
4 2 10 0
5 2 10 10
5 2 10 0
5 2 10 2
6 0 0 0
0 0 0 0