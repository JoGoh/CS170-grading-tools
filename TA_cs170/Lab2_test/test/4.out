AddToFront : List1 ( 1 nodes):   21
AddToEnd   : List1 ( 2 nodes):   21  94
Find       : Value 21 was found in List1.
Find       : Value 1 was not found in List1.
Find       : Value 94 was found in List1.
Delete     : List1 ( 1 nodes):   21
Delete     : List1 ( 1 nodes):   21
Delete     : List1 ( 1 nodes):   21
Delete     : List1 ( 0 nodes): 
AddToFront : List2 ( 1 nodes):   10
AddToEnd   : List2 ( 2 nodes):   10  10
Find       : Value 10 was found in List2.
Delete     : List2 ( 1 nodes):   10
Delete     : List2 ( 0 nodes): 
Insert     : List2 ( 1 nodes):   10
Insert     : List2 ( 2 nodes):   10  10
Insert     : List2 ( 3 nodes):   10  10  10
Concatenate: 
List1 ( 3 nodes):   10  10  10
List2 ( 0 nodes): 
